ipet/evaluation/IPETEvalTable.py

```diff
@@ -829,7 +909,8 @@ def reduceToColumns(self, df_long : DataFrame) -> DataFrame:
         additionalfiltercolumns = [afc for afc in additionalfiltercolumns if afc not in set(usercolumns + neededcolumns)]
         result = df_long.loc[:, usercolumns + neededcolumns + additionalfiltercolumns + self.countercolumns]
         self.usercolumns = usercolumns
-        return result
+        
+        return df_target
```

Comment:
As it is we can now remove lines 896 to 911. Is this what you intended? How do you take care of the columns that are needed for the filters (but that are not used in the columns)?